frappe/www/printview.py

```diff
@@ -136,8 +136,14 @@ def get_template_from_string():
 
 	letter_head = frappe._dict(get_letter_head(doc, no_letterhead) or {})
 
+	if 'content' in letter_head:
```

Comment:
should be if `letter_head.content` - letter_head will have a property `content` which can be None